CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package vcs2l
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.1.1 (2025-07-30)
+------------------
+* Changed base python version to 3.5. (`#21 <https://github.com/ros-infrastructure/vcs2l/pull/21>`_)
+* Release 1.1.1 (`#22 <https://github.com/ros-infrastructure/vcs2l/pull/22>`_)
```

Comment:
It's not often necessary to add the release commit PR to the changelog but is ultimately harmless.